### DISCUSSION TITLE
v6.0.1

### DISCUSSION THREAD

**Discussion by tay1orjones** [IBM/plex#442] [General]
# Changes in 6.0.1

## [Unreleased]
- IBM Plex Sans CN

### Fixed
- Incorrect shape for U+08A4 (ARABIC LETTER FEH WITH DOT BELOW AND THREE DOTS ABOVE) and related glyphs
  referenced in [issue 429](https://github.com/IBM/plex/issues/429)

### Fixed
- Outline interpolation errors in glyphs /uni95B9 and /asterisk.rotat
  referenced in [issue 404](https://github.com/IBM/plex/issues/404)
  referenced in [issue 405](https://github.com/IBM/plex/issues/405)
- Small errors in head table
- Small errors in hhea table

<hr /><em>This discussion was created from the release <a href='https://github.com/IBM/plex/releases/tag/v6.0.1'>v6.0.1</a>.</em>

**Comment by BoldMonday** [comment]
Thanks for the new release!

There was also an update we did to Plex Sans KR last year that has not been released yet.
The updated files can be found in Box. Please let me know if you need a direct link to them.

## [IBM PLEX SANS KR V1.2] - 2021-09-13
### Changed
- abbreviated font names used for style mapping (name ID1 + ID2) to improve interoperability
